src/sass/layout/_header.scss

```diff
@@ -17,6 +17,10 @@
     padding-bottom: 73px;
     padding-left: 80px;
     padding-right: 80px;
+
```

Comment:
Please also do the same current page highlight to the footer

Reply:
done